test/integ/conftest.py

```diff
@@ -1,16 +1,15 @@
 #  Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I suggest to add absolute import in all files to avoid import inconsistencies.
We can use flake8 future import to validate it like in the python sdk https://github.com/aws/sagemaker-python-sdk/blob/master/tox.ini#L27

Reply:
good idea. since we use neither flake nor tox in this repo currently, however, I'll save it for a different PR to include TC configuration, etc.

Reply:
sounds good.